src/ClientInterface.php

```diff
@@ -146,29 +147,14 @@ public function userReport(string $report_uid,
     /**
      * Make report.
      *
-     * @param string            $report_type_uid Unique report type ID (e.g.: `some_report_uid` or
-     *                                           `some_report_uid@domain`)
-     * @param string            $type            Request type (e.g.: `VIN`, `GRZ`, `STS`, `PTS`, `CHASSIS`, etc.)
-     * @param string            $value           Request body (e.g.: `Z94CB41AAGR323020` (VIN), `А111АА177` (GRZ))
-     * @param array<mixed>|null $options         Additional request options
-     * @param bool|null         $is_force        Force update report, if it already was generated previously
-     * @param string|null       $on_update       Call (using `post` method) when report content updated
-     * @param string|null       $on_complete     Call (using `post` method) when report generation completed
-     * @param array<mixed>|null $data            Additional request data
+     * @param ReportMakeParams $report_make_params Object with data to make report
      *
      * @throws BadRequestException
      * @throws BadResponseException
      *
      * @return UserReportMakeResponse
      */
-    public function userReportMake(string $report_type_uid,
-                                   string $type,
-                                   string $value,
-                                   ?array $options = [],
-                                   ?bool $is_force = false,
-                                   ?string $on_update = null,
-                                   ?string $on_complete = null,
-                                   ?array $data = null): UserReportMakeResponse;
+    public function userReportMake(ReportMakeParams $report_make_params): UserReportMakeResponse;
```

Comment:
```suggestion
    public function userReportMake(ReportMakeParams $params): UserReportMakeResponse;
```

Reply:
Thanks! I use `$params` in `Client.php` but forgot to rename here.

Fixed here and in doc-block 4e7c8623be12362d0aa57cb77d5c4715417225dc